roofit/roofitcore/src/RooAbsPdf.cxx

```diff
@@ -3190,13 +3152,10 @@ RooPlot* RooAbsPdf::paramOn(RooPlot* frame, const RooAbsData* data, const char *
 /// values specify the initial relative position of the text box in the plot frame.
 
 RooPlot* RooAbsPdf::paramOn(RooPlot* frame, const RooArgSet& params, bool showConstants, const char *label,
```

Comment:
Here you are changing the `paramOn` interface. Can the old one be kept for backward compatibility and mark as deprecated and maybe removed later on ?

Reply:
This is `private` function that is an implementation detail which is called by the public `paramOn()` function. So it doesn't matter if the change the interface

Reply:
Good! I missed it was a private function.